components/automate-cli/pkg/verifyserver/utils/fiberutils/os_test.go

```diff
@@ -12,3 +12,42 @@ func TestExecuteShellCommand(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "Hello World\n", string(out))
 }
+
+func TestCheckPath(t *testing.T) {
+	type args struct {
+		cmd string
+	}
+	tests := []struct {
+		description      string
+		args             args
+		expectedResponse string
+		expectedError    string
+	}{
+		{
+			description: "If the cammand is present on the path",
+			args: args{
+				cmd: "mkdir",
+			},
+			expectedResponse: "/bin/mkdir",
+			expectedError: "",
+		},
+		{
+			description: "If the cammand is not present on the path",
+			args: args{
+				cmd: "wrongcammand",
+			},
+			expectedResponse: "",
+			expectedError: `exec: "wrongcammand": executable file not found in $PATH`,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.description, func(t *testing.T) {
+			got, err := fiberutils.CheckPath(tt.args.cmd)
+			if (err != nil) {
```

Comment:
you need to assert the test based on your inputs, here you are depending on the error returned while executing the test